csrc/runtime/executor_utils.h

```diff
@@ -253,12 +233,5 @@ void validateCircularBuffering(
     kir::Kernel* kernel,
     ExpressionEvaluator& expr_eval);
 
-//! Query the target GPU version number NVRTC compiles CUDA kernels for
```

Comment:
Moved to compiled_kernel.cpp

Reply:
Actually these are likely just removed as they should now be self contained in runtime/compiled_kernel.cpp